src/sentry/relay/config.py

```diff
@@ -397,6 +400,47 @@ def _filter_option_to_config_setting(flt, setting):
 )
 
 
+class _TransactionThreshold(TypedDict):
+    metric: Optional[str]  # Either 'duration' or 'lcp'
+    threshold: float
+
+
+class _TransactionThresholdConfig(TypedDict):
+    projectThreshold: _TransactionThreshold
+    transactionThresholds: Mapping[str, _TransactionThreshold]
```

Comment:
what's this mapping? What are the keys?

Reply:
Keys are transaction names. So each transaction can have its own threshold defined.